varnish/tox.ini

```diff
@@ -0,0 +1,40 @@
+[tox]
+minversion = 2.0
+basepython = py27
+envlist =
+    flake8
+    unit
+    varnish4
+    varnish5
+
+[testenv]
+platform = linux|darwin|win32
+deps =
+    ../datadog_checks_base
+    -rrequirements-dev.txt
+passenv =
+    DOCKER*
+    COMPOSE*
+commands =
+    pip install --require-hashes -r requirements.txt
+    pytest -m"integration" -v
+
+[testenv:varnish4]
+setenv = VARNISH_VERSION=4.1.7
+
+[testenv:varnish5]
+setenv = VARNISH_VERSION=5.2.1
+
+[testenv:unit]
+commands = 
+	pytest -m"not integration" -v
```

Comment:
Since this overrides `commands` from `[testenv]` we'll need to either:

a. put `pip install --require-hashes -r requirements.txt` here, or
b. move `pytest -m"integration" -v` from `[testenv]` to the other envs and put `{[testenv]deps}` in all

I like `b`